instrumentation/events/lttng-module/net.h

```diff
@@ -8,7 +8,106 @@
 #include <linux/skbuff.h>
```

Comment:
oh no I'm sorry, I pushed all the fixes on the other lttng-modules-dev repo (the copy of compudj's lttng-modules-dev), so I just did the same pull request as before. My bad...